### ISSUE TITLE
Testing effort

### ISSUE THREAD

**Issue by animeshsingh** [IBM/kubernetes-container-service-gameon-java-microservice#5]
@Tomcli Please help get this Journey tested

**Comment by Tomcli** [comment]
GameOn works on my cluster without any problem. However, I think we should add some instructions on how to delete the GameOn deployments in README.

**Comment by animeshsingh** [comment]
Thanks @Tomcli  
cc @AnthonyAmanse for cleanup

**Comment by animeshsingh** [comment]
@AnthonyAmanse can we do one test run on Minikube as well.

**Comment by animeshsingh** [comment]
Issue with Kubes 

Tim Robinson 
[7:49 PM] 
looking at the GameOn journey WIP… worked well using Deploy Toolchain button. But not so good with minikube on my Mac on 0.18.0 (latest). The problem I saw was that kubectl get nodes came back with a name instead of public IP of the cluster. All went good until trying to start the core services. Lots of
```time="2017-05-05T01:55:39Z" level=info msg="Starting NGINX" 
[ALERT] 124/015539 (10) : parsing [/etc/haproxy/haproxy-dev.cfg:97] : 'server room1' : invalid address: 'minikube' in 'minikube:30980'

[ALERT] 124/015539 (10) : parsing [/etc/haproxy/haproxy-dev.cfg:110] : 'server map1' : invalid address: 'minikube' in 'minikube:30947'

```
messages popping up. So I deleted it all and changed the replace IP script with the docker-machine ip of 192.168.99.100 and re-ran. Then it was happier. Things seem to start. However, the application seems to have problems working correctly. Not getting good information when moving to a room, unable to move back to the starting room. Were you getting better results with minikube?

[7:52] 
some minor easy fixes I’d suggest. add an s (silent) option to these commands to just return the response codes:

[7:52] 
`curl -w "%{http_code}" "<Public IP of your cluster>:31200/health" -o /dev/null`

[7:52] 
should be

[7:53] 
`curl -sw "%{http_code}" "<Public IP of your cluster>:31200/health" -o /dev/null`

[7:55] 
I’d suggest to move to the teardown instructions the `kubectl delete pod setup` command. That way folks that use deploy to toolchain button will see it when doing the other cleanup.

[7:56] 
final thing… drop the “s” from Dockers in the Reference item.
